core/src/main/java/org/elasticsearch/action/count/CountRequestBuilder.java

```diff
@@ -162,16 +106,6 @@ protected CountResponse getDelegatedFromInstigator(SearchResponse response) {
 
     @Override
     public String toString() {
-        if (sourceBuilder != null) {
-            return sourceBuilder.toString();
-        }
-        if (request.source() != null) {
-            try {
-                return XContentHelper.convertToJson(request.source().toBytesArray(), false, true);
-            } catch (Exception e) {
-                return "{ \"error\" : \"" + ExceptionsHelper.detailedMessage(e) + "\"}";
-            }
-        }
-        return new QuerySourceBuilder().toString();
+        return request.sourceBuilderString();
```

Comment:
why not using CountRequest#toString here? then CountRequest#sourceBuilderString could go away.